src/Parser/Monk/Brewmaster/Modules/Core/StaggerFabricator.js

```diff
@@ -0,0 +1,104 @@
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { GIFT_OF_THE_OX_SPELLS } from '../../Constants';
+
+const PURIFY_BASE = 0.4;
+const ELUSIVE_DANCE_PURIFY = 0.2;
+const STAGGERING_AROUND_PURIFY = 0.01;
+const ISB_QUICK_SIP_PURIFY = 0.05;
+const T20_4PC_PURIFY = 0.05;
+const debug = false;
+
+export const EVENT_STAGGER_POOL_UPDATE = "stagger_pool_update";
+export const KIND_STAGGER_ADDED = "added";
+export const KIND_STAGGER_REMOVED = "removed";
+
+/**
+ * Fabricate events corresponding to stagger pool updates. Each stagger
+ * tick, purify, and stagger absorb generates one event.
+ */
+class StaggerFabricator extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  _has_t20_4pc = false;
+  _PURIFY_AMOUNT = PURIFY_BASE;
+  _stagger_pool = 0;
```

Comment:
Using a private property for this while making it public is unnecessary, please just call this `staggerPool`.

Reply:
The idea with that was to (strongly) indicate that although you can *read* the value, it should not be modified except within this module. A comment may suffice for that, though.